src/20_Components/amp-social-share.html

```diff
@@ -18,6 +18,11 @@
     Ensure you've configured your canonical URL.
   -->
   <link rel="canonical" href="<%host%>/components/amp-social-share/">
+  <!--
+    Origin trial tokens for `ampbyexample.com` and `*.cdn.ampproject.org`. (Needed for `type=system`; [see below](#native-share-dialog-some-browsers-).)
+  -->
+  <meta http-equiv="origin-trial" data-feature="Web Share" data-expires="2017-03-06" content="AtEHR9Ke52xpgqsGbRi+U1kDOYSDQTfzeITL+Q/nvBiOx/qCW/vMWQxyt1y8+6688/9pjfdnCc0sJBqgkwQkjwsAAABLeyJvcmlnaW4iOiJodHRwczovL2JlZWJvLnJlZDo0NDMiLCJmZWF0dXJlIjoiV2ViU2hhcmUiLCJleHBpcnkiOjE0ODg4MTA2NjR9">
```

Comment:
Waiting on a token for `ampbyexample.com`; should be ready in a day or two. Unfortunately they only last a couple of months.